netebpfext/ebpf_ext_attach_provider.c

```diff
@@ -0,0 +1,185 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation
// SPDX-License-Identifier: MIT
```
Please use the above style per https://github.com/microsoft/ebpf-for-windows/blob/master/docs/DevelopmentGuide.md#license-header

Same comment on other files